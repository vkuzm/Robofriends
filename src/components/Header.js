import React from 'react';

class Header extends React.Component {
	shouldComponentUpdate() {
		return false;
	}

	render() {
		return <h1 className='f2'>Robofriends</h1>
	}
}

export default Header;