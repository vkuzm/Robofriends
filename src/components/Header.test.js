import React from 'react';
import Header from './Header';
import { shallow } from 'enzyme';
import toJson from 'enzyme-to-json';

it('Header component renders without crashing', () => {
	const shallowComponent = toJson(shallow(<Header />));
  expect(shallowComponent).toMatchSnapshot();
});