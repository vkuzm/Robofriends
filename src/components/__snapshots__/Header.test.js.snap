// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Header component renders without crashing 1`] = `
<h1
  className="f2"
>
  Robofriends
</h1>
`;
